libpromises/cf3parse.y

```diff
@@ -674,14 +674,6 @@ constraint:            constraint_id                        /* BUNDLE ONLY */
                                            cp->offset.end = P.offsets.current;
                                            cp->offset.context = P.offsets.last_class_id;
                                            P.currentstype->offset.end = P.offsets.current;
-
-                                           // Cache whether there are subbundles for later $(this.promiser) logic
-
-                                           if (strcmp(P.lval,"usebundle") == 0 || strcmp(P.lval,"edit_line") == 0
-                                               || strcmp(P.lval,"edit_xml") == 0 || strcmp(P.lval,"home_bundle") == 0)
-                                           {
-                                               P.currentpromise->has_subbundles = true;
-                                           }
```

Comment:
No, that's ok, I noticed now that it's a separate commit, that made things clear.